### ISSUE TITLE
Accessibility issues in Consumers Page

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#800]
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds

Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast-enhanced?application=axeAPI

[
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#1a202c",
                    "bgColor": "#c6841b",
                    "contrastRatio": 5.21,
                    "fontSize": "9.6pt (12.8px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<span class=\"chakra-avatar css-khdfw0\"><div role=\"img\" aria-label=\"Mark F Mark L\" class=\"chakra-avatar__initials css-1pd2iu3\">MF</div></span>",
                        "target": [
                            ".chakra-avatar"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 5.21 (foreground color: #1a202c, background color: #c6841b, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<div role=\"img\" aria-label=\"Mark F Mark L\" class=\"chakra-avatar__initials css-1pd2iu3\">MF</div>",
        "target": [
            ".chakra-avatar__initials"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 5.21 (foreground color: #1a202c, background color: #c6841b, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 7:1"
    },
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#1a5a96",
                    "bgColor": "#f1f1f1",
                    "contrastRatio": 6.3,
                    "fontSize": "10.5pt (14px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<a class=\"chakra-breadcrumb__link css-5yp7oh\" href=\"/manager/namespaces\">Namespaces (newplatform)</a>",
        "target": [
            ".css-5yp7oh"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
    },
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#606060",
                    "bgColor": "#ffffff",
                    "contrastRatio": 6.28,
                    "fontSize": "9.8pt (13px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<div class=\"css-a1k8dr\">",
                        "target": [
                            ".css-a1k8dr"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.28 (foreground color: #606060, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<p class=\"chakra-text css-1awll3g\">Filtered tags will appear here</p>",
        "target": [
            ".css-1awll3g"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.28 (foreground color: #606060, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 7:1"
    }
]

![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/3d8ec859-866d-41cd-83e1-0bc12a55a4fd)

*************************************************************************************
